src/date-util/date-util.ts

```diff
@@ -376,6 +376,11 @@ export namespace DateUtil {
       });
     return format12HourInLocale(formatResult, opts.locale);
   }
+
+  export function isValidLastDay(d: Date): boolean {
+    const date = new Date(d.getFullYear(), d.getMonth(), d.getDate() + 1);
+    return date.getDate() === 1;
+  }
```

Comment:
1. 
```suggestion
  export function isLastDateOfMonth(d: Date): boolean {
    const date = new Date(d.getFullYear(), d.getMonth(), d.getDate() + 1);
    return date.getDate() === 1;
  }
```
2. d의 타입이 DateType이 아닌 Date인 것이 맞나요?
3. 특별한 이점이 있거나 그런 거는 아니지만 lastDayOfMonth을 이용했어도 되겠습니다. (그러고보니 이것도 getLastDateOfMonth가 더 적절한 이름이겠네요)

Reply:
4. beginOfMonth, endOfMonth, lastDayOfMonth 시리즈와 같이 붙어있는게 낫겠습니다

Reply:
해당 함수를 만들었던 이유는.. 매월 말일에 실행되어야 하는 배치를 만들어야 할때 실행된 날이 말일인지 확인이 필요하여 작성하였습니다.

2.넵. 의도라고 한다면 배치에서 매일 실행 될 때 그날의 날짜가 말일인 것을 체크해야 하기 때문에 `new Date()` 를 사용할 것이라 타입을 Date만 하였습니다.
3. lastDayOfMonth를 이용하여 +1일을 추가해 비교하는 것도 가능하겠군요!
4. 위치를 옮기도록 하겠습니다.

Reply:
2. 그 날짜 데이터가 반드시 Date여야 하느냐, string으로 들어와도 상관 없는 것 아니냐인 것이죠. (number로 넘겨주는 경우는 잘 없을테니)

Reply:
말일인지 확인하려는 거니까 isLateDateOfMonth가 맞는 이름이 아니냐는 겁니다.
valid는 유효하다 아니다의 의미라서 부적절한 이름입니다.
그리고 날짜는 date입니다. day와 date의 의미는 좀 달라요

Reply:
string으로 해도 상관은 없기에, DateType으로 변경해서 사용 할 수 있도록  하는것이 좋겠네요! 🙏

Reply:
기존 세 함수(beginOfMonth, endOfMonth, lastDayOfMonth)에 대한 이름 변경은 제가 따로 할테니 우선 그냥 두세요